app/views/layouts/application.html.erb

```diff
@@ -1,9 +1,6 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <% if current_user&.admin? %>
-      <%= link_to "Admin Dashboard", admin_root_path %>
```

Comment:
Why isn't this needed anymore?

Reply:
Makes total sense!

It was my bad -- I knew I reviewed before you were done.

Reply:
No not at all!  I appreciate you taking a look.